lc0/src/chess/board.h

```diff
@@ -115,7 +115,7 @@ class ChessBoard {
 
   BitBoard ours() const { return our_pieces_; }
   BitBoard theirs() const { return their_pieces_; }
-  BitBoard pawns() const { return pawns_ * kPawnMask; }
+  BitBoard pawns() const;
```

Comment:
This function won't be able to inline, without linker-level inlining, which not all compilers have. It's fine to keep it like that though, I guess, should not have any noticeable performance impact.